apis/src/api/v1/auth/get_token_handler.rs

```diff
@@ -0,0 +1,78 @@
+use super::encode::{jwt_encode, Bot};
+use super::jwt_secret::JwtSecret;
+use actix_web::post;
+use actix_web::web::{Data, Json};
+use actix_web::HttpResponse;
+use argon2::{
+    password_hash::{PasswordHash, PasswordVerifier},
+    Argon2,
+};
+use db_lib::get_conn;
+use db_lib::{models::User, DbPool};
+use serde_json::json;
+
+#[post("/api/v1/auth/token")]
```

Comment:
So everyone can use the API to play as a bot ? (I don't see any filtering of bot user accounts). But I understand that this PR is a stepping stone.

Reply:
@PenguinWithATie suggested to add a column to `User`. I think that's what I am going to do before shipping it.
We also need the `bot` field to advertise bot challenges and to indicate that an account is a bot.